### PR TITLE
health#provider: fix duplicated output/stderr

### PR DESCRIPTION
**Pull request by blueyed**
Ref: https://github.com/neovim/neovim/pull/11047#issuecomment-532268826

### CODE REVIEW

File: runtime/autoload/health/provider.vim [hunk 1]
```diff
@@ -89,8 +90,15 @@ function! s:system(cmd, ...) abort
...
-          \ jobid, s:shell_error, s:shellify(a:cmd), string(getcwd()), opts.output, opts.stderr))
+    let error = printf("Command error (job=%d, exit code %d): `%s` (in %s)",
+          \ jobid, s:shell_error, s:shellify(a:cmd), string(getcwd()))
+    if !empty(opts.output)
+      let error .= "\nOutput: " . opts.output
+    end
+    if !empty(opts.stderr)
+      let error .= "\nStderr: " . opts.stderr
+    end
+    call health#report_error(error)
```
Comment (justinmk) [inline runtime/autoload/health/provider.vim]: generally the word "error" should be avoided, I would call it `err` or `e`.

File: runtime/autoload/health/provider.vim [hunk 2]
```diff
@@ -89,8 +90,15 @@ function! s:system(cmd, ...) abort
...
   elseif s:shell_error != 0 && !ignore_error
-    call health#report_error(printf("Command error (job=%d, exit code %d): `%s` (in %s)\nOutput: %s\nStderr: %s",
-          \ jobid, s:shell_error, s:shellify(a:cmd), string(getcwd()), opts.output, opts.stderr))
+    let error = printf("Command error (job=%d, exit code %d): `%s` (in %s)",
+          \ jobid, s:shell_error, s:shellify(a:cmd), string(getcwd()))
+    if !empty(opts.output)
+      let error .= "\nOutput: " . opts.output
+    end
+    if !empty(opts.stderr)
+      let error .= "\nStderr: " . opts.stderr
```
Comment (justinmk) [inline runtime/autoload/health/provider.vim]: it is weird to title-case "Stderr". And  "Output" for that matter. This is not a formal letter.

**Review by justinmk — APPROVED** [review]

File: runtime/autoload/health/provider.vim (hunk 1 above)
Comment (blueyed) [reply runtime/autoload/health/provider.vim]: Thanks for telling me again.. :)
Used "emsg".

File: runtime/autoload/health/provider.vim (hunk 2 above)
Comment (blueyed) [reply runtime/autoload/health/provider.vim]: True.

**Pull request merged**